### DISCUSSION TITLE
Welcome to emmairwin Discussions!

### DISCUSSION THREAD

**Discussion by emmairwin** [emmairwin/emmairwin#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emmairwin** [comment]
test

**Comment by Rocin1738** [comment]
Hello, pleasure meeting you.

**Comment by Rocin1738** [comment]
I need some help pulling a request on my project "README.md" What suggestions would point me in the right direction.
